### PR TITLE
Fixes rank iteration

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes rank iteration

Before:

issue

After:

Screen Shot 2023-01-09 at 3 25 24 PM

fixes https://github.com/EveripediaNetwork/issues/issues/948

### CODE REVIEW

**Comment by Adebesin-Cell** [comment]
I’ll increase the font sizes then. 👍

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 I've adjusted the font sizes.

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
The font size looks okay. 
Can't wait to finish the new ranking page. 
I think that would be so much more better.

also, why is the nft list not up to ten. It doesn't look good that way.
image

**Comment by Adebesin-Cell** [comment]
> The font size looks okay. Can't wait to finish the new ranking page. I think that would be so much more better.
> 
> also, why is the nft list not up to ten. It doesn't look good that way. image

Ashaka said the backend is filtering NFt wikis without a 24h volume. For the first page autoglyphs is an example of such nft wiki.

**Comment by Simplypheyie96** [comment]
> > The font size looks okay. Can't wait to finish the new ranking page. I think that would be so much more better.
> > also, why is the nft list not up to ten. It doesn't look good that way. image
> 
> Ashaka said the backend is filtering NFt wikis without a 24h volume. For the first page autoglyphs is an example of such nft wiki.

hmmnn.
you can go ahead and merge then.

**Comment by Adebesin-Cell** [comment]
Yeah, I think we’ll review more on that 🙂.

File: src/components/Rank/InvalidRankCardItem.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const InvalidRankCardItem = ({
           >
             <Text
               color="inactiveText"
-              fontSize={{ md: 'xs', base: 'sm' }}
+              fontSize={{ base: 'xs' }}
```
Comment (Royal-lobster) [inline src/components/Rank/InvalidRankCardItem.tsx:47]:
```suggestion
              fontSize="xs"
```
I think this is same as 'xs'

File: src/components/Rank/InvalidRankCardItem.tsx [hunk 2]
```diff
@@ -57,8 +57,7 @@ export const InvalidRankCardItem = ({
               width="100%"
               justifyContent="end"
             >
-              {/* <AiFillCaretUp color={upIndicationIconColor} /> */}
-              <Text fontWeight="bold" fontSize={{ md: 'xs', base: 'xs' }}>
+              <Text fontWeight="bold" fontSize={{ base: 'xs' }}>
```
Comment (Royal-lobster) [inline src/components/Rank/InvalidRankCardItem.tsx:60]:
```suggestion
              <Text fontWeight="bold" fontSize="xs" >
```
please check if it is same, if it is same we can directly use "xs".

Comment (Adebesin-Cell) [reply src/components/Rank/InvalidRankCardItem.tsx:60]: Alright, I’ll look into i!t.

**Comment by Adebesin-Cell** [comment]
@Simplypheyie96 I think you can check now, we have the 10 rankings listed out properly.

**Review by Softdev1 — APPROVED** [review]

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96 I think you can check now, we have the 10 rankings listed out properly.

good to go

**Pull request merged**